disclosure_wrangler.py

```diff
@@ -178,4 +233,23 @@ def lambda_handler(event, context):
             return {"success": False, "error": error_message}
 
     logger.info("Successfully completed module: " + current_module)
+
     return {"success": True, "checkpoint": checkpoint}
+
+
+def invoke_method(lambda_execution_name, payload, lambda_client):
```

Comment:
This looks useful enough to make its way to aws_funks in future.

Reply:
Make a tech-debt jira ticket for it, looks like we are going to be short on tickets in sprint soon.